standalone-node/installation_scripts/user-apps/readme.md

```diff
@@ -12,3 +12,398 @@ To copy,configure, or launch your applications, use the custom `cloud-init` sect
 - Store your application files in this folder.
 - Update the `cloud-init` section as needed to automate deployment.
 
+
```

Comment:
I think this section also we need to move it to standalone-node/docs/user-guide folder as a new md. file... this folder will not be read by end user. The README here is a placeholder so that the folder is not empty.